bin/announce.js

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
This should be in `tools`, not `bin`, unless I'm misunderstanding what this is used for.  `bin` is where binaries we expect consumers to use go.

Reply:
oh.

traditionally `bin/` is for any script that's outside the build process, and `tools/` is for any script that's inside the build process

i can move it.  that means i need to move run, too.

frankly, under this definition, the only thing that should be in `bin` is the resulting `peggy` instance.  marking node scripts which aren't meant to be consumed by the outside world as binaries is an anti-pattern.  it has security implications.  runnable scripts should just be run directly, rather than shebanged and permissioned.

Reply:
I'm ok with the tooling scripts being called with `node` rather than being shebanged and permissioned.

Reply:
This will be continued with #126

Reply:
Tool is moved